backend/plugins/loyalty_api/src/modules/score/db/models/ScoreLog.ts

```
@@ -103,13 +111,13 @@ export const loadScoreLogClass = (models: IModels, subdomain: string) => {
     }
 
     public static async getScoreLogs(doc: IScoreLogParams) {
-      const { stageId, number, orderType } = doc;
+      const { stageId, pipelineId, boardId, number, orderType } = doc;
 
       const filter = await generateFilter(doc, models, subdomain);
 
       let filterAggregate: any[] = [];
```

ESLint

[error] 'filterAggregate' is never reassigned. Use 'const' instead.

(prefer-const)